bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -137,14 +139,19 @@ templatef = h5py.File(args.bank_file, 'r')
 chisq = trigf[args.ifo+'/chisq'][:]
 chisq_dof = trigf[args.ifo+'/chisq_dof'][:]
 rchisq = chisq / (2 * chisq_dof - 2)
+try:
+    sgchisq = trigf[args.ifo+'/sg_chisq'][:]
+except:
+    sgchisq = None
```

Comment:
It will blow up at you. I think that's actually the right behavior since it will tell you it is missing this information, and then you can figure out why.